### PR TITLE
az.json update

### PR DESCRIPTION
**Pull request by hs-qwerty**
update

### CODE REVIEW

File: locales/az/az.json [hunk 1]
```diff
@@ -539,7 +539,7 @@
     "Show Recovery Codes": "Bərpa Kodlarını Göstər",
     "Showing": "Anasini",
     "Sierra Leone": "Sierra-Leone",
-    "Signed in as": "Signed in as",
+    "Signed in as": "kimi daxil olub",
```
Comment (andrey-helldar) [inline locales/az/az.json:542]: Is it correct that the sentence starts with a lowercase letter?

File: locales/az/az.json [hunk 2]
```diff
@@ -589,7 +589,7 @@
     "Thailand": "Tayland",
     "Thanks for signing up! Before getting started, could you verify your email address by clicking on the link we just emailed to you? If you didn't receive the email, we will gladly send you another.": "Qeydiyyatdan keçdiyiniz üçün təşəkkür edirik! Siz başlamaq əvvəl, Biz yalnız e-poçt vasitəsilə göndərilir linki tıklayarak e-poçt ünvanınızı təsdiq edə bilər? Bir məktub almadıysanız, biz məmnuniyyətlə Başqa bir şey göndərəcəyik.",
     "Thanks for your continued support. We've attached a copy of your invoice for your records. Please let us know if you have any questions or concerns.": "Thanks for your continued support. We've attached a copy of your invoice for your records. Please let us know if you have any questions or concerns.",
-    "Thanks,": "Thanks,",
+    "Thanks,": "təşəkkürlər,",
```
Comment (andrey-helldar) [inline locales/az/az.json:592]: Is it correct that the first character of a word is lowercase?

Comment (hs-qwerty) [reply locales/az/az.json:592]: I will correct my mistake and send

Comment (andrey-helldar) [reply locales/az/az.json:592]:
```suggestion
    "Thanks,": "Təşəkkürlər,",
```

File: locales/az/az.json (hunk 1 above)
Comment (andrey-helldar) [reply locales/az/az.json:542]:
```suggestion
    "Signed in as": "Kimi daxil olub",
```

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Təşəkkürlər

**Pull request merged**